### PR TITLE
Update example project

### PR DESCRIPTION
**Pull request by TheMutsi**
## Goal

Update Example project

### CODE REVIEW

**Review by priettt — APPROVED** [review]

File: examples/react-native-test-suite/App.js [hunk 1]
```diff
@@ -1,15 +1,102 @@
...
 // import codePush from 'react-native-code-push';
 
+import {
+  initialize,
+  endAppStartup,
+  setJavaScriptBundlePath,
+} from '@embrace-io/react-native';
+
 import MainNavigation from './src/navigation/MainNavigation';
+// import EmbraceApolloLink from '@react-native-embrace/apollo-graphql/lib/index';
```
Comment (jpmunz) [inline examples/react-native-test-suite/App.js:12]: is the commented code here needed for reference? Can it be removed?

File: examples/react-native-test-suite/android/log.txt
Comment (jpmunz) [inline examples/react-native-test-suite/android/log.txt:1]: should these log.txt files be git ignored?

File: examples/react-native-test-suite/ios/EmbraceTestSuite/AppDelegate.mm [hunk 2]
```diff
@@ -17,8 +17,7 @@ - (BOOL)application:(UIApplication *)application
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
 
-  
-  [[Embrace sharedInstance] startWithLaunchOptions:launchOptions framework:EMBAppFrameworkReactNative];
+  //  [[Embrace sharedInstance] startWithLaunchOptions:launchOptions framework:EMBAppFrameworkReactNative];
```
Comment (jpmunz) [inline examples/react-native-test-suite/ios/EmbraceTestSuite/AppDelegate.mm:20]: not needed?

File: examples/react-native-test-suite/src/screens/HomeScreen.js [hunk 3]
```diff
@@ -116,6 +300,7 @@ const HomeScreen = () => {
       <WebViewScreen visible={showWebView} closePopup={setShowWebView} />
       {hasMomentStarted && (
         <View style={{position: 'absolute', bottom: 10, width: '100%'}}>
+          {asdf}
```
Comment (jpmunz) [inline examples/react-native-test-suite/src/screens/HomeScreen.js:303]: typo? I don't see this defined

File: examples/react-native-test-suite/src/screens/HomeScreen.js
Comment (jpmunz) [inline examples/react-native-test-suite/src/screens/HomeScreen.js:1]: similar question for these screens are the commented sections used during testing or can they be removed?

File: examples/react-native-test-suite/android/app/google-services.json
Comment (jpmunz) [inline examples/react-native-test-suite/android/app/google-services.json:1]: double checking that the keys in here + appcenter-config.json files are ok to expose

File: examples/react-native-test-suite/android/app/release/app-release.aab
Comment (jpmunz) [inline examples/react-native-test-suite/android/app/release/app-release.aab:1]: should this be git ignored?

**Comment by jpmunz** [comment]
Closing as the changes here are a bit stale now

**Pull request closed without merging**